smart-contract/assembly/contracts/internals/file-list.ts

```diff
@@ -0,0 +1,34 @@
+import { FILES_PATH_LIST } from '../utils/const';
+
+export function _pushFilePath(filePath: string): void {
+  const filePathList = FILES_PATH_LIST.mustValue();
+  filePathList.push(filePath);
+  FILES_PATH_LIST.set(filePathList);
+  _getFilePathList();
+}
+
+export function _removeFilePath(filePath: string): void {
+  const filePathList = FILES_PATH_LIST.mustValue();
+  if (!_isPathFileInList(filePath)) return;
+  // TODO: check if it works
+  FILES_PATH_LIST.set(filePathList.filter((path) => path !== filePath));
```

Comment:
```suggestion
  FILES_PATH_LIST.set(filePathList.filter((path) => path !== filePath));
```
this doesn't compile when called: 

```ERROR AS100: Not implemented: Closures
    :
 14 │ FILES_PATH_LIST.set(filePathList.filter((path) => path !== filePath));
    │                                                            ~~~~~~~~
    └─ in assembly/contracts/internals/file-list.ts(14,62)```

Reply:
see: https://www.assemblyscript.org/status.html

Reply:
ok thanks I fix it

Reply:
I made the fix, I will let you try